common4j/src/main/com/microsoft/identity/common/java/util/HashMapExtensions.java

```diff
@@ -56,6 +59,8 @@ public static HashMap<String, String> getJsonResponse(HttpResponse webResponse)
      */
     public static HashMap<String, String> getJsonResponseFromResponseBody(String responseBody) throws JSONException {
         final HashMap<String, String> response = new HashMap<>();
+        Span span = Span.current();
```

Comment:
nit: final

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1847/commits/9bc9320913bc03bd1a733bdf0fd5e8442caafae0